MatrixSDK/Background/Store/SyncResponseFileStore/MXSyncResponseFileStore.swift

```diff
@@ -183,6 +183,13 @@ public class MXSyncResponseFileStore: NSObject {
         metadata.outdatedSyncResponseIds.removeAll(where: { $0 == id })
         saveMetaData(metadata)
     }
+
+    private func deleteSyncResponseIds(ids: [String]) {
```

Comment:
Could the `deleteSyncResponseId` (delete single ID), now internally call `deleteSyncResponseIds(ids: [id])` to avoid code duplication?

Reply:
Yes surely